lib/client.js

```diff
@@ -80,56 +81,70 @@ class Metacom extends EventEmitter {
     };
   }
 
-  async message(data) {
-    if (data === '{}') return;
+  async message(msg) {
```

Comment:
Then we'll have a naming collision, since `protocol.deserialize` returns `{ type: string; data: { ... } }`.
So we either leave it as `msg` or need to change `data` property of deserialize return interface `deserialize`.